packages/client/src/pages/AppPage.tsx

```diff
@@ -61,7 +64,7 @@ const AppPage = (props: { children: React.ReactNode }) => {
       <NotificationSnackbar />
       <LoadWebappInjection>{props.children}</LoadWebappInjection>
       <InviteToast />
-      <Debug />
+      {debugModeEnabled && <Debug />}
```

Comment:
i don't think we want a debug mode at all. maybe instead of disabling it entirely, we can opt in via a console command, as it can be very useful for debugging bugs as we find them in deployments